Doc/library/py_compile.rst

```diff
@@ -42,6 +42,13 @@ byte-code cache files in the directory containing the source code.
    is raised.  This function returns the path to byte-compiled file, i.e.
    whatever *cfile* value was used.
 
+   The doraise and quiet arguments determine how errors are handled while
```

Comment:
doraise -> \*doraise\*

quiet -> \*quiet\*

Same for other usages of doraise and quiet.